### PR TITLE
🎉 Add web-ui for mint-nft

### PR DESCRIPTION
**Pull request by williamchong**
image

### CODE REVIEW

File: web-ui/pages/mint-nft/index.vue [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+      </div>
+      <br>
+      <br>
+      or
+      <br>
+      <div>
+        <p><label>Upload ISCN data json (iscn.json) file: </label></p>
+        <input type="file" @change="onISCNFileChange">
+        <br>
+        <button :disabled="isLoading" @click="onISCNFileInput">
```
Comment (nwingt) [inline web-ui/pages/mint-nft/index.vue:29]:
```suggestion
        <button :disabled="isLoading" style="margin-top: 16px" @click="onISCNFileInput">
```

File: web-ui/layouts/default.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+</template>
+<style>
+body {
+    margin: 0;
+}
+
+h1 {
+    display: inline-block;
+}
+
```
Comment (nwingt) [inline web-ui/layouts/default.vue:23]: Strange to make header inline
```suggestion

```

File: web-ui/layouts/default.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    border-collapse: collapse;
+    border: 1px solid;
+}
+
+footer {
+    padding: 12px 12px 20px;
+
+    display: flex;
+    gap: 10px;
+}
```
Comment (nwingt) [inline web-ui/layouts/default.vue:36]: Seem footer is not used?
```suggestion
```

File: web-ui/pages/mint-nft/index.vue [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+      <h2>3. Mint ISCN</h2>
+      <div>
+        <label>Number of NFT to mint:</label>
+        <input v-model="nftMintCount" type="number">
+        <p><label>Upload NFT default data json (nfts_default.json) file: </label></p>
+        <input type="file" @change="onMintNFTDefaultFileChange">
+        <p><label>Upload NFT CSV (nfts.csv) file: </label></p>
+        <input type="file" @change="onMintNFTFileChange">
+        <br>
+        <button :disabled="isLoading" @click="onMintNFTStart">
```
Comment (nwingt) [inline web-ui/pages/mint-nft/index.vue:91]:
```suggestion
        <button :disabled="isLoading" style="margin-top: 16px" @click="onMintNFTStart">
```

File: web-ui/pages/mint-nft/index.vue [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+const nftMintCount = ref(100)
+const nftData = ref<any>(null)
+const nftCSVData = ref('')
+
+const iscnId = computed(() => iscnData.value?.['@id'])
+const classId = computed(() => classData.value?.id)
+
+onMounted(async () => {
+
+})
```
Comment (nwingt) [inline web-ui/pages/mint-nft/index.vue:153]: What is this for?

Comment (williamchong) [reply web-ui/pages/mint-nft/index.vue]: Placeholder that wasn't used, will remove

**Review by nwingt — APPROVED** [review]

**Pull request merged**